### PR TITLE
Add setup & run instructions, technical stack and branches to readme

### LINKED ISSUE #26: Bug: Add technical stack and setup and run instructions, branches in readme
### Describe the bug

As this project is now actively developed, we should add technical stack and setup instructions in readme

### To Reproduce
1. Go to https://github.com/anitab-org/stem-diverse-tv-cms
2. Scroll down to readme.md

### Expected behavior

Technical stack and setup instructions should be added in readme.md

### Screenshots

stem

### Desktop (please complete the following information):
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by Rahulm2310**
### Description

Add setup & run instructions, technical stack and branches info to readme.md.
Fixes #26 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Rahulm2310** [comment]
@annabauza Please review this PR. Thanks 🙂

**Review by annabauza — CHANGES_REQUESTED** [review]
I have left you small comments :)

File: README.md [hunk 1]
```diff
@@ -67,12 +67,24 @@ upstream        https://github.com/anitab-org/stem-diverse-tv-cms (fetch)
...
 
 **This project is under active development**
 
 Please read our [Contributing Guidelines](docs/contributing_guidelines.md), [Code of Conduct](docs/code_of_conduct.md) and [Reporting Guidelines](docs/reporting_guidelines.md) thoroughly.
 
+## Branches
+- master: This contains the latest code. All the contributing PRs must be sent to this branch.
+- production: Merge master into production triggers deployment of the website.
 
 ## Contact
```
[the remark this answers is not in the document]
Comment (Rahulm2310) [reply README.md:89]: @annabauza  Isn't it already added below?

[the remark this answers is not in the document]
Comment (annabauza) [reply README.md:89]: yes you're right :)

**Comment by Rahulm2310** [comment]
@annabauza Added the suggested changes and some badges. Please review it 🙂

**Review by annabauza — APPROVED** [review]

**Pull request merged**